build_rpm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+git checkout -- setup.cfg
+rm -rf build dist *.egg *.egg-info
+
+# Build community source tarball
+
+python setup.py sdist --formats=gztar
+
+# Build F5 rpm spec
+
+/bin/cp -f f5-setup.cfg setup.cfg
+
+python setup.py bdist_rpm --spec-only \
+  --release=1 \
+  --provides=f5-neutron-lbaas-dashboard \
+  --packager="Qin Zhao <q.zhao@f5.com>" \
+  --post-install=rpm_post_install.sh
+
+# Repackage source tarball
+
+cd dist
+
+VERSION=$(rpmspec -q --qf "%{version}" f5-neutron-lbaas-dashboard.spec)
+
+OLD=neutron-lbaas-dashboard-${VERSION}
+NEW=f5-neutron-lbaas-dashboard-${VERSION}
+
+tar -zxf ${OLD}.tar.gz
+
+mv ${OLD} ${NEW}
+
+cd ${NEW}
+
+/bin/cp -f f5-setup.cfg setup.cfg
+
+mv neutron_lbaas_dashboard/enabled/_1481_project_ng_loadbalancersv2_panel.py neutron_lbaas_dashboard/enabled/_1491_project_ng_loadbalancersv2_panel.py
+
+mv neutron_lbaas_dashboard f5_neutron_lbaas_dashboard
+
+rm -rf neutron_lbaas_dashboard.egg-info
+
+cp -rp ../../f5_neutron_lbaas_dashboard.egg-info ./
```

Comment:
One of the benefits for using absolute path in script is when we need to debug the script with 'sh -x', it's very easy to navigate to each (called) file. Relative path is not so clear where and where.. also not convenient to debug..